test/validator/validation_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'minitest_helper'
+
+module CollectionJson
+  class Serializer
+    class Validation
+      class TestValidation < MiniTest::Test
+        include TestHelper
+
+        def test_that_instantiating_validation_raises_error
+          user = User.new(name: 'Carles Jove')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.